src/main/resources/avro/rnaquantificationmethods.avdl

```diff
@@ -1,207 +1,146 @@
-
-  /**
-  Quantification Groups of interest.
-  */
-  union { null, string } quantificationGroupId = null;
-
-  /**
-  Specifies the maximum number of results to return in a single page.
-  If unspecified, a system default will be used.
-  */
-  union { null, int } pageSize = null;
-
-  /**
-  The continuation token, which is used to page through large result sets.
-  To get the next page of results, set this parameter to the value of
-  'nextPageToken' from the previous response.
-  */
-  union { null, string } pageToken = null;
+// This is the response from 'POST /expressionlevel/search' expressed as JSON.
+message SearchExpressionLevelResponse {
+  // The list of matching quantifications.
+  repeated ExpressionLevel expression_level = 1;
+
+  // The continuation token, which is used to page through large result sets.
+  // To get the next page of results, set this parameter to the value of
+  // 'nextPageToken' from the previous response.
+  string next_page_token = 2;
 }
 
-/**
-This is the response from 'POST /quantificationgroup/search' expressed as JSON.
-*/
-record SearchQuantificationGroupResponse {
-  /** The list of matching quantification groups. */
-  array<org.ga4gh.models.QuantificationGroup> quantificationGroup = [];
-
-  /**
-  The continuation token, which is used to page through large result sets.
-  To get the next page of results, set this parameter to the value of
-  'nextPageToken' from the previous response.
-  */
-  union { null, string } nextPageToken = null;
+// ****************  /quantificationgroup/search  *******************
+// This request maps to the body of 'POST /quantificationgroup/search'
+// as JSON.
+message SearchQuantificationGroupRequest {
+  // RNA Quantification to search.
+  string rna_quantification_id = 1;
+
+  // Quantification Groups of interest.
+  string quantification_group_id = 2;
+
+  // Specifies the maximum number of results to return in a single page.
+  // If unspecified, a system default will be used.
+  int32 page_size = 3;
+
+  // The continuation token, which is used to page through large result sets.
+  // To get the next page of results, set this parameter to the value of
+  // 'nextPageToken' from the previous response.
+  string page_token = 4;
 }
 
-/**
-Gets a list of 'QuantificationGroup' matching the search criteria.
-
-'POST /quantificationgroup/search' must accept JSON version of
-'SearchQuantificationGroupRequest' as the post body and will return a JSON
-version of 'SearchQuantificationGroupResponse'.
-*/
-SearchQuantificationGroupResponse searchQuantificationGroup(
-    /**
-    This request maps to the body of 'POST /quantificationgroup/search'
-    as JSON.
-    */
-    SearchQuantificationGroupRequest request) throws GAException;
+// This is the response from 'POST /quantificationgroup/search' expressed as JSON.
+message SearchQuantificationGroupResponse {
+  // The list of matching quantification groups.
+  repeated QuantificationGroup quantification_group = 1;
```

Comment:
Not directly related to proto, but these repeated response fields are plural in the other APIs, i.e. quantification_groups